### PR TITLE
feat: Add new error template with a nonce and custom image

### PR DESCRIPTION
**Pull request by sveneberth**
Template originally designed by @icubes, customized by me and finally improved by @timmytiefkuehl.

### CODE REVIEW

File: src/viur/core/request.py [hunk 1]
```diff
@@ -357,10 +359,8 @@ def _process(self):
                     if filename := conf["viur.mainApp"].render.getTemplateFileName((f"{error_info['status']}", "error"),
                                                                                    raise_exception=False):
                         template = conf["viur.mainApp"].render.getEnv().get_template(filename)
-                        res = template.render(error_info)
-
-                        # fixme: this might be the viur/core/template/error.html ...
-                        extendCsp({"style-src": ['sha256-Lwf7c88gJwuw6L6p6ILPSs/+Ui7zCk8VaIvp8wLhQ4A=']})
+                        res = template.render(error_info, nonce=(nonce := utils.generateRandomString(16)))
```
Comment (ArneGudermann) [inline src/viur/core/request.py:362]:
```suggestion
                        nonce = utils.generateRandomString(16)
                        res = template.render(error_info, nonce=nonce)
```
I know this work but i think this more readable.

File: src/viur/core/request.py [hunk 2]
```diff
@@ -357,10 +359,9 @@ def _process(self):
...
                         template = conf["viur.mainApp"].render.getEnv().get_template(filename)
-                        res = template.render(error_info)
-
-                        # fixme: this might be the viur/core/template/error.html ...
-                        extendCsp({"style-src": ['sha256-Lwf7c88gJwuw6L6p6ILPSs/+Ui7zCk8VaIvp8wLhQ4A=']})
+                        nonce = utils.generateRandomString(16)
+                        res = template.render(error_info, nonce=nonce)
+                        extendCsp({"style-src": [f"nonce-{nonce}"]})
                     else:
                         res = f"""<html><h1>{error_info["status"]} - {error_info["reason"]}"""
```
Comment (sveneberth) [inline src/viur/core/request.py:366]: This isn't valid HTML.

Take look at https://gist.github.com/cmalven/1885287 for minimal examples. 

I want to rework this part too

**Comment by phorward** [comment]
Can this be turned into "mergable" soon?

File: src/viur/core/request.py (hunk 2 above)
Comment (sveneberth) [reply src/viur/core/request.py]: um yeah. There I forgot to pay attention to my own reminder. Thank you, I have done it

**Review by phorward — APPROVED** [review]

**Review by ArneGudermann — APPROVED** [review]

**Review by phorward — APPROVED** [review]

**Pull request merged**